crates/ruma-signatures/Cargo.toml

```diff
@@ -13,8 +13,14 @@ version = "0.7.1"
 
 [dependencies]
 base64 = "0.13.0"
+# ed25519-dalek = "1.0.1"
+ed25519-dalek = { path = "/home/jonathan/dev/ed25519-dalek" }
+pkcs8 = { git = "https://github.com/ShadowJonathan/utils", branch = "oak", features = ["alloc"] }
```

Comment:
This PR will be blocked until `pkcs8` releases a new version which includes `OneAsymmetricKey`, and support for parsing it.

Reply:
I'll also remove those `path =` dependencies when the PR is unblocked too.